packages/hardhat/contracts/strategy/CrabStrategy.sol

```diff
@@ -31,11 +31,12 @@ contract CrabStrategy is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Ownab
     using StrategyMath for uint256;
     using Address for address payable;
 
-    uint32 public constant TWAP_PERIOD = 420 seconds;
+    /// @dev the TWAP_PERIOD used in the PowerPerp Controller contract
     uint32 public constant POWER_PERP_PERIOD = 420 seconds;
-    // strategy will only allow hedging if collateral to trade is at least 0.1% of the total strategy collateral
-    uint256 public constant DELTA_HEDGE_THRESHOLD = 1e15;
+    /// @dev twap period to use for hedge calculations
+    uint32 public twapPeriod = 420 seconds;
```

Comment:
Can u move this down just next to the non-immutable vars ?

Reply:
only reason was packing the uint32s, is that helpful here or not?

Reply:
changed name

Reply:
yup